easybuild/easyconfigs/l/libdierckx/libdierckx-1993_O2_fPIC.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru libdierckx-master-orig/Makefile libdierckx-master/Makefile
```

Comment:
Please include a description and author information in the patch. This will require that the checksum is updated in the easyconfig.

Reply:
Okay, this is now done.